nuclia.md

```diff
@@ -0,0 +1,85 @@
+# Replacing Sphinx Search with Nuclia Search in Plone
+
+## Overview
+
+This documentation outlines the process of replacing Sphinx search in a Plone site with Nuclia search. The goal is to explain the approach taken and provide instructions for replicating the process in another project.

```

Comment:
This rewrite elaborates what you have done, why it is awesome, and serves as an introduction. I also added some [MyST syntax](https://6.docs.plone.org/contributing/documentation/myst-reference.html) that uses arbitrary cross-references to a [page](https://6.docs.plone.org/contributing/documentation/myst-reference.html#link-to-a-chapter-or-page) and using [Intersphinx](https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#cross-project-intersphinx-links).

We prefer one sentence per line. See https://6.docs.plone.org/contributing/documentation/authors.html#plone-documentation-styleguide

```suggestion
This documentation outlines the process of replacing the default Sphinx search in documentation websites with Nuclia search.

With Sphinx search, you are limited to searching only that one site.
Nuclia's "knowledge box" can index content from multiple websites and display results across them.

The implementation described below displays search results across the {doc}`Plone 6 Documentation </index>` and `Training <inv:training:std:doc:index>` websites.
You can follow and replicate this process for your projects.
```